publish/publish.proj

```diff
@@ -134,14 +134,17 @@
   </Target>
 
   <Target Name="PublishCoreHostPackagesToFeed"
```

Comment:
nit: I think that `PublishCoreHostPackagesToFeed` and `PushCoreHostPackagesToFeed` is (only slightly) confusing.  What about naming this `Publish` and `PublishCoreHostPackagesToFeed`?